src/cmd/package.go

```diff
@@ -143,6 +144,23 @@ func newPackageCreateCommand(v *viper.Viper) *cobra.Command {
 	return cmd
 }
 
+// Converts registry overrides to a structured type.
+// Input is of the following form:
+// []string{"docker.io/library=docker.example.com", "docker.io=docker.example.com"}
+func parseRegistryOverrides(overrides []string) ([]types.RegistryOverride, error) {
+	result := make([]types.RegistryOverride, len(overrides))
+	for i, override := range overrides {
+		source, override, found := strings.Cut(override, "=")
```

Comment:
this `override` variable returned from `Cut()` shadows the original override loop variable. I would delineate these from eachother